packages/jira-adapter/src/filters/forms/forms.ts

```diff
@@ -120,7 +143,19 @@ const filter: FilterCreator = ({ config, client, fetchQuery }) => ({
       })))
       .flat()
       .filter(isDefined)
-    forms.forEach(form => elements.push(form))
+    forms.forEach(form => {
+      fixFormsKeysName({
+        instance: form,
+        transformKeyFunc: ((key, value) => {
+          if (naclCase(key) !== key) {
+            const newKey = naclCase(key)
+            value[newKey] = value[key]
+            delete value[key]
+          }
```

Comment:
I think that it's better to use the calculated value and not recalculate it.
Same goes for the `invertNaclCase`
```suggestion
         const naclCaseKey = naclCase(key)
          if (naclCaseKey !== key) {
            value[naclCaseKey] = value[key]
            delete value[key]
          }
```